.github/workflows/on-push-main.yml

```diff
@@ -0,0 +1,45 @@
+name: On push main
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  sonarqube:
+    name: SonarQube Trigger
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources 🔰
+        uses: actions/checkout@v4
+        with:
+          # Disabling shallow clone is recommended for improving relevancy of reporting
+          fetch-depth: 0
+
+      - name: Setup Node.js 🧮
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: Cache Node.js modules 💾
+        uses: actions/cache@v3
```

Comment:
there is already a newer version available: https://github.com/actions/cache/releases

Reply:
Good hint thank you! the actions are now included in the dependabot

Reply:
nice! :+1: